packages/@pollyjs/adapter-fetch/tests/integration/adapter-test.js

```diff
@@ -103,6 +104,39 @@ describe('Integration | Fetch Adapter', function() {
     expect(error.message).to.contain('The user aborted a request.');
   });
 
+  it('should be able to download binary content', async function() {
+    this.timeout(10000);
+
+    const fetch = async () =>
+      Buffer.from(
+        await this.fetch('https://placekitten.com/32/32').then(res =>
```

Comment:
Same comment in previous PR, would like if we can lean on our own test server vs external APIs

Reply:
Yeah, I'll add a binary route to our middleware to serve an image.